### PR TITLE
Multiple doc updates

### PR DESCRIPTION
**Pull request by Aaryan-R-S**
### Fixes Issue: [LIT-2536](https://linear.app/litprotocol/issue/LIT-2536/multiple-minor-docs-update)

# Description

Lots of points in this section of the Notion page: https://www.notion.so/litprotocol/Simplify-Mainnet-Migration-abb502a527144ec5b2aecc63d247215f?pvs=4#8b51454a526d48459b6927987467d813

Trying to update the docs for it!

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Introducing new feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

Link to the relevant updated sections in the Netlify preview

- [ ] Link 1: [description of change]
- [ ] Link 2: [...]

# Checklist:

General
- [ ] I have performed a self-review of my code
- [ ] I have fixed all grammar issues (can use an AI tool to check), and explanations are in active voice
- [ ] I have checked the additions are concise
- [ ] Language is consistent with existing documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published (ie. SDK changes, node dependencies)

If I have added a new concept, I have
- [ ] included a beginner friendly explanation
- [ ] included a basic technical introduction and code sample
- [ ] new terms are defined, both in relevant new pages and in the glossary

### CODE REVIEW

File: docs/concepts/capacity-credits-concept.md [hunk 1]
```diff
@@ -30,13 +30,13 @@ For minting a Capacity Credits NFT see example docs for using our contract-sdk [
...
+Alice can create a session capability object that specifies the ability to Authenticate with an Capacity Credits NFT as well as request for Threshold Execution (collect and combine key shares from the Lit nodes above the threshold to form the complete signature after executing Lit Action and getting responses) against a particular Lit Action IPFS CID(s). Alice then signs and issues these capabilities to Bob.
 
 Bob can generate an `AuthSig` by delegating equal rights to Bob's session keys, and attaching the capabilities granted to him by Alice as a proof in the session object. Bob can subsequently generate a `SessionSig` that requests for Alice's Capacity Credits NFT, specifying the Lit Action IPFS CID in the `resourceAbilityRequests` field.
 
 
 ### **Best Practices**
 
 - **Capacity Management**: Keep an eye on your usage limit and expiration date.
-- **Understanding Limits**: Be aware of the free tier rate limit. Capacity credits can be used to reserve more usage.
+- **Understanding Limits**: Be aware of the free tier rate limit which is set to 3 free requests per day as of now on `Manzano` and `Habanero` networks. Capacity credits can be used to reserve more usage.
```
Comment (Adarsh-Kumar28) [inline docs/concepts/capacity-credits-concept.md:41]: There are no free requests on Habenaro: https://www.notion.so/litprotocol/Simplify-Mainnet-Migration-abb502a527144ec5b2aecc63d247215f?pvs=4#f4c39ea75c9c48c3894df4fa01d36890

File: docs/network/migration-guide.md [hunk 2]
```diff
@@ -23,12 +23,10 @@ If you’re migrating from Jalapeno to Habanero, it’s important to remember th
 
 ### For encryption / decryption use cases
 
-Cayenne and Habanero have different root keys, so you’ll need to re-encrypt your user’s content in order to migrate to Habanero
+Cayenne and Habanero have different root keys, so you’ll need to re-encrypt your user’s content in order to migrate to Habanero. In order to perform re-encryption, you can use the same v3 SDK for both Cayenne and Habanero, but note that you will need 2 instances of the SDK, one connected to each network. First, use Cayenne SDK instance to decrypt the encrypted data and then use Habanero SDK instance to re-encrypt it.
```
Comment (Adarsh-Kumar28) [inline docs/network/migration-guide.md:26]: Also add Manzano

File: docs/sdk/authentication/session-sigs/get-session-sigs.md [hunk 3]
```diff
@@ -70,6 +70,7 @@ const sessionSigs = await litNodeClient.getSessionSigs({
     }
   ],
   authNeededCallback,
+  capacityDelegationAuthSig,
```
Comment (Adarsh-Kumar28) [inline docs/sdk/authentication/session-sigs/get-session-sigs.md:73]: Where is this defined?

File: docs/concepts/capacity-credits-concept.md
Comment (Adarsh-Kumar28) [inline docs/concepts/capacity-credits-concept.md:1]: Add about the Explorer which can be used to mint CC. Also link to the minting CC section

File: docs/concepts/capacity-credits-concept.md [hunk 4]
```diff
@@ -30,13 +30,13 @@ For minting a Capacity Credits NFT see example docs for using our contract-sdk [
...
 
-Alice can create a session capability object that specifies the ability to Authenticate with an Capacity Credits NFT as well as request for Threshold Execution against a particular Lit Action IPFS CID(s). Alice then signs and issues these capabilities to Bob.
+Alice can create a session capability object that specifies the ability to Authenticate with an Capacity Credits NFT as well as request for Threshold Execution (collect and combine key shares from the Lit nodes above the threshold to form the complete signature after executing Lit Action and getting responses) against a particular Lit Action IPFS CID(s). Alice then signs and issues these capabilities to Bob.
 
 Bob can generate an `AuthSig` by delegating equal rights to Bob's session keys, and attaching the capabilities granted to him by Alice as a proof in the session object. Bob can subsequently generate a `SessionSig` that requests for Alice's Capacity Credits NFT, specifying the Lit Action IPFS CID in the `resourceAbilityRequests` field.
```
Comment (Adarsh-Kumar28) [inline docs/concepts/capacity-credits-concept.md:35]: Should be "Alice can generate" not "Bob can generate"

File: docs/concepts/capacity-credits-concept.md [hunk 5]
```diff
@@ -30,13 +30,13 @@ For minting a Capacity Credits NFT see example docs for using our contract-sdk [
 ### **Delegating Capacity — Paying for Your Users’ Requests**
 You can also delegate your capacity credits to other users. For example, Alice owns a Capacity Credit NFT and wants to let Bob use it, but only for a specific Lit Actions or another resource or set of resources that she owns.
 
-Alice can create a session capability object that specifies the ability to Authenticate with an Capacity Credits NFT as well as request for Threshold Execution against a particular Lit Action IPFS CID(s). Alice then signs and issues these capabilities to Bob.
+Alice can create a session capability object that specifies the ability to Authenticate with an Capacity Credits NFT as well as request for Threshold Execution (collect and combine key shares from the Lit nodes above the threshold to form the complete signature after executing Lit Action and getting responses) against a particular Lit Action IPFS CID(s). Alice then signs and issues these capabilities to Bob.
```
Comment (Adarsh-Kumar28) [inline docs/concepts/capacity-credits-concept.md:33]:
```suggestion
Alice can create a session capability object that specifies the ability to Authenticate with an Capacity Credits NFT as well as request for Threshold Execution, for example, against a particular Lit Action IPFS CID(s). Alice, then signs and issues these capabilities to Bob.
```

File: docs/migration/overview.md [hunk 6]
```diff
@@ -14,11 +14,11 @@ The **Lit JS SDK V3** replaces the existing access control condition based encry
 
 :::caution
 
-The Lit development team will not be migrating the access control conditions that have been "stored" in the `jalapeno` and `serrano` networks but we will continue to maintain support for them for a finite period of time.
+The Lit development team will not be migrating the access control conditions that have been "stored" in the `jalapeno` and `serrano` networks but we will continue to maintain support for them for a finite period of time. Therefore, we will request users to migrate to `habanero` and `manzano` networks.
 
 :::
 
-If you wish to continue using these access control conditions for encryption or signing JWTs, please continue using the V2 SDK.
+If you wish to continue using these access control conditions for encryption or signing JWTs, please continue using the V2 SDK. But still we will recommend users to migrate to V3 SDK as it supports our new networks, `habanero` and `manzano`, which are not supported by V2 SDK.
```
Comment (Adarsh-Kumar28) [inline docs/migration/overview.md:21]:
```suggestion
If you wish to continue using these access control conditions for encryption or signing JWTs, please continue using the V2 SDK. But we strongly recommend users to migrate to V3 SDK as it supports our new networks, `habanero` and `manzano`, which are not supported by V2 SDK.
```

**Review by Adarsh-Kumar28 — CHANGES_REQUESTED** [review]
Missing banner: https://www.notion.so/litprotocol/Simplify-Mainnet-Migration-abb502a527144ec5b2aecc63d247215f?pvs=4#186e581a33534e49945e75c23011f486

File: docs/concepts/capacity-credits-concept.md (hunk 1 above)
Comment (Aaryan-R-S) [reply docs/concepts/capacity-credits-concept.md:41]: Done @Adarsh-Kumar28 :)

File: docs/network/migration-guide.md (hunk 2 above)
Comment (Aaryan-R-S) [reply docs/network/migration-guide.md:26]: Done @Adarsh-Kumar28 :)

File: docs/sdk/authentication/session-sigs/get-session-sigs.md (hunk 3 above)
Comment (Aaryan-R-S) [reply docs/sdk/authentication/session-sigs/get-session-sigs.md:73]: Done @Adarsh-Kumar28 :)

File: docs/concepts/capacity-credits-concept.md
Comment (Aaryan-R-S) [reply docs/concepts/capacity-credits-concept.md:1]: Done @Adarsh-Kumar28 :)

File: docs/concepts/capacity-credits-concept.md (hunk 4 above)
Comment (Aaryan-R-S) [reply docs/concepts/capacity-credits-concept.md:35]: Ah right! Done @Adarsh-Kumar28 :)

File: docs/concepts/capacity-credits-concept.md (hunk 5 above)
Comment (Aaryan-R-S) [reply docs/concepts/capacity-credits-concept.md:33]: Done @Adarsh-Kumar28 :)

File: docs/migration/overview.md (hunk 6 above)
Comment (Aaryan-R-S) [reply docs/migration/overview.md:21]: Done @Adarsh-Kumar28 :)

**Review by Adarsh-Kumar28 — APPROVED** [review]
LGTM!

**Review by a1ttech — APPROVED** [review]

**Pull request merged**